ibis/backends/tests/test_client.py

```diff
@@ -1379,6 +1379,10 @@ def test_persist_expression_repeated_cache(alltypes):
     reason="mssql supports support temporary tables through naming conventions",
 )
 @mark.notimpl(["exasol"], reason="Exasol does not support temporary tables")
+@mark.broken(
+    ["oracle"],
+    reason="Among other reasons, oracles errors don't inherit from Exception",
+)
```

Comment:
Oh boy.

Was this working before (and xfailing)?

Reply:
hmm, I thought I removed that comment, must have reapplied it during a rebase.  I think I'm wrong about this.

Reply:
this was working before because sqlalchemy intercepted the oracle error and included the name of the missing table/view.  `oracledb` raises an error that simply states that the table/view is unknown but doesn't include the name of the table the user tried to access.  for now, just marking as `notimpl` until we decide if want to wrap and reraise with more info.